ember-cli-build.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-env node */
+
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+const configFunc = require('./config/environment');
+
+const nonCdnEnvironments = ['development', 'test'];
+
+const {
+    EMBER_ENV
+} = process.env;
+
+module.exports = function(defaults) {
+    const config = configFunc(EMBER_ENV);
+    const useCdn = !nonCdnEnvironments.includes(EMBER_ENV);
+
+    const css = {
+        'app': '/assets/reviews-service.css'
+    };
+
+
+    const {
+        OSF: {url: osfUrl}
+    } = defaults.project.config(EMBER_ENV);
+
+    // Reference: https://github.com/travis-ci/travis-web/blob/master/ember-cli-build.js
+    const app = new EmberApp(defaults, {
+        sourcemaps: {
+            enabled: true,
+            extensions: ['js']
+        },
+        vendorFiles: {
+            // next line is needed to prevent ember-cli to load
+            // handlebars (it happens automatically in 0.1.x)
+            'handlebars.js': {production: null},
+            [useCdn ? 'ember.js' : '']: false,
+            [useCdn ? 'jquery.js' : '']: false,
+        },
+        'ember-bootstrap': {
+            importBootstrapCSS: false
+        },
+        // Needed for branded themes
+        fingerprint: {
+            customHash: config.ASSET_SUFFIX
+        },
+        outputPaths: {
+            app: {
+                css
+            }
+        },
+        sassOptions: {
+            includePaths: [
+                'node_modules/@centerforopenscience/ember-osf/addon/styles',
+                'bower_components/bootstrap-sass/assets/stylesheets',
+                'bower_components/osf-style/sass',
+                'bower_components/hint.css',
+                'bower_components/c3',
+                'bower_components/bootstrap-daterangepicker',
+            ]
+        },
+        inlineContent: {
+            raven: {
+                enabled: useCdn,
+                content: `
+                    <script src="https://cdn.ravenjs.com/3.5.1/ember/raven.min.js"></script>
+                    <script>Raven.config("${config.sentryDSN}", {}).install();</script>`
+            },
+            cdn: {
+                enabled: useCdn,
+                content: `
+                    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+                    <script src="//cdnjs.cloudflare.com/ajax/libs/ember.js/2.7.1/ember.prod.js"></script>`
```

Comment:
@jamescdavis can verify this, but we should be using `ember.min.js` rather than `ember.prod.js`